src/main/java/pl/stormit/tinyurl/domain/model/UrlAnalytics.java

```diff
@@ -0,0 +1,37 @@
+package pl.stormit.tinyurl.domain.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import java.time.Instant;
+
+@Entity
+@Table (name = "url_analytics")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+public class UrlAnalytics {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Think about UUID as id type.

Reply:
I changed id type to UUID.
I used Long type, because we have this type in Url Entity.
Shouldn't we also do a refactoring on the Url Entity with a change of type to UUID?
Bartek, what do you think?
So that everything is consistent.

Reply:
Okey, let's change on UUID